File: Producto/routes-service/src/main/java/com/tagok/routes_service/dto/response/portico/TramoResponse.java
package com.tagok.routes_service.dto.response.portico;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TramoResponse
{
    private String entrada;
    private String nombreEntrada;
    private String salida;
    private String nombreSalida;

    private List<ReglaTarifariaResponse> reglas;

    private CalendarioTarifarioResponse calendario;
}

File: Producto/tag-ok-app/app/src/main/java/com/tagok/app/data/dto/portico/TramoResponse.kt
package com.tagok.app.data.dto.portico

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class TramoResponse(
    val entrada: String,
    val salida: String,
    val reglas: List<ReglaTarifariaResponse>,
    val calendario: CalendarioTarifarioResponse)
